### DISCUSSION TITLE
Trouble in accepting the assignment 1. Running the Demo

### DISCUSSION THREAD

**Discussion by Taylorwutianyi** [AccelerationConsortium/ac-microcourses#137] [Hello World]
<img width="1050" alt="Screenshot 2024-12-27 at 3 51 51 PM" src="https://github.com/user-attachments/assets/333137e2-a176-4908-906c-d208296a5530" />

Refreshed the page several times, but the error existed

**Comment by sgbaird** [comment]
Hi @Taylorwutianyi , sorry for the delay here. I think this should be resolved now (I made a change to this assignment's visibility, not realizing at the time it would have this effect, and have since changed it back). Could you try again?

**Comment by Taylorwutianyi** [comment]
Happy New Year, Sterling! Thank you so much! It works now!
